tsconfig.json

```diff
@@ -3,12 +3,16 @@
         "lib": ["es2015", "es2016.array.include", "es2017.object"],
         "noEmit": true,
         "target": "esnext",
+        "module": "commonjs",
+        "moduleResolution": "node",
         "strict": true,
         "allowJs": true,
         "checkJs": true
     },
     "types": ["node"],
     "include": [
+        "./dev-tools/global.d.ts",
+        "./dev-tools/node.d.ts",
```

Comment:
I think it might be best just to `// @ts-ignore` every line that sets `global.Dex` and `global.toId`, rather than needing an entire `.d.ts` file for it...

Reply:
I prefer this way because it preserves the <code>require()</code> checking, but it's up to you.

Reply:
`dev-tools/node.d.ts` doesn't seem to have any `require()`s to be checked?

Reply:
Sorry, I meant the <code>require()</code>s when setting up globals.